gptcache/manager/vector_data/manager.py

```diff
@@ -86,6 +86,17 @@ def get(name, **kwargs):
                 collection_name=collection_name,
                 top_k=top_k,
             )
+        elif name == "hnswlib":
+            from gptcache.manager.vector_data.hnswlib_store import Hnswlib
+
+            dimension = kwargs.get("dimension", DIMENSION)
+            index_path = kwargs.pop("index_path", "./hnswlib_index.bin")
+            max_elements = kwargs.pop("max_elements")
```

Comment:
should associate with the current `max_size` param

Reply:
According to the current interface design, users can directly create a vector store, so it is not possible to associate max_size with it.